src/streamlink/plugins/orf_tvthek.py

```diff
@@ -47,6 +47,7 @@ def _get_streams(self):
             except KeyError:
                 continue
             stream = HLSStream.parse_variant_playlist(self.session, url)
+            self.session.new_http_session()
```

Comment:
why not just `self.session.http.close()` or `http.close()`

---

do you have an example url where the video will only work with this?
or are this the geo-blocked videos